docs/resources/repository.md

```diff
@@ -71,9 +71,9 @@ resource "nexus_repository" "docker_group" {
 
 The following arguments are supported:
 
-* `format` - (Required, ForceNew) Repository format
+* `format` - (Required, ForceNew) Repository format. Possible values: `apt`, `bower`, `conan`, `docker`, `gitlfs`, `go`, `helm`, `maven2`, `npm`, `nuget`, `p2`, `pypi`, `raw`, `rubygems`, `yum`.
 * `name` - (Required) A unique identifier for this repository
-* `type` - (Required, ForceNew) Repository type
+* `type` - (Required, ForceNew) Repository type. Possible values: `group`, `hosted`, `proxy`
```

Comment:
please also change it in resource_repository.go file. we generate this docs.

Reply:
But looks like it is exist there:
https://github.com/datadrivers/terraform-provider-nexus/blob/master/nexus/resource_repository.go#L88
and
https://github.com/datadrivers/terraform-provider-nexus/blob/master/nexus/resource_repository.go#L106
It took values from nexus.RepositoryFormats and nexus.RepositoryTypes

Reply:
https://github.com/datadrivers/go-nexus-client/blob/v0.19.1/repository.go#L36
and
https://github.com/datadrivers/go-nexus-client/blob/v0.19.1/repository.go#L55

Reply:
ha, i see, that `make docs` do not cover it

Reply:
fixed in https://github.com/datadrivers/terraform-provider-nexus/pull/116/commits/8446152765c81da2c7f40ad2b38ed08f23b86379